public/stylesheets/main.css

```diff
@@ -37,3 +41,26 @@ h3 {
 }
 
 /* TODO - Add your own styles here */
+
+<<<<<<< HEAD
+
+
+
+
+.profile-section{
+  background-color: #B4DAD7;
+}
+
+.box{
+  background-color: #77BCB4;
+}
+
+.box:nth-of-type(1) {
+  background-color: #55A4AA; 
+    -webkit-column-count: 2; /* Chrome, Safari, Opera */
+    -moz-column-count: 2; /* Firefox */
+    column-count: 2;
+}
+=======
+
+>>>>>>> origin/master
```

Comment:
These symbols refer to merge conflicts... make sure you don't commit them! Read up on the git manual about resolving conflicts.